tests/utils/mock_microgrid.py

```diff
@@ -49,6 +49,12 @@ def __init__(self, components: Set[Component], connections: Set[Connection]):
         meter_channels = self._create_meter_channels()
         ev_charger_channels = self._create_ev_charger_channels()
 
+        self._all_channels: Dict[int, Broadcast[Any]] = {}
+        self._all_channels.update(bat_channels)
+        self._all_channels.update(inv_channels)
+        self._all_channels.update(meter_channels)
+        self._all_channels.update(ev_charger_channels)
```

Comment:
Not sure what happened with my pro-tip (maybe I forgot to submit?) but here it goes again:

```suggestion
        self._all_channels: Dict[int, Broadcast[Any]] = {
		**bat_channels,
		**inv_channels,
		**meter_channels,
		**ev_charger_channels,
	}
```

(just a suggestion)